pkg/build/starnix.go

```diff
@@ -17,6 +17,11 @@ import (
 
 type starnix struct{}
 
+// The location of the default ffx build config (a Fuchsia implementation detail),
+// relative to the kernel build dir. Contains default relative paths of build
+// artifacts.
+const ffxBuildConfig = "ffx-config.json"
```

Comment:
Do you know if there is a future plan for avoiding relying on this implementation detail?

Reply:
That's a good question. The short answer from the `ffx` folks was "no" -- they're in sort of a similar situation to us, since they're in the SDK and rely on this hard-coded path from the Fuchsia tree. I do think it might make sense for `ffx` to provide a `--default-env` or `--default-build-config` option (or some such name) that would grab the config from this path, and am planning to raise that question again.